### PR TITLE
V-0.0.1

### PR DESCRIPTION
**Pull request by YoseptF**
homepage

---

In this project we made a webapp where you can add a list of the books you've read

---

## Local installation

1. Clone this repo
```
$ git clone git@github.com:YoseptF/library.git
``` 
2. install the dependencies
```
$ npm install || yarn install
```
3. Run the index.html file (We recomend using [Live server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) if you're already using vscode)

4. Open your localhost
```
localhost:5500 || 127.0.0.1:5500
```

### CODE REVIEW

**Comment by YoseptF** [comment]
# JS Library new local testing process

---

## Local installation

1. Clone this repo
```
$ git clone git@github.com:YoseptF/library.git
``` 
2. cd into the correct branch
```
$ cd [your desired branch]
``` 
2. install the dependencies
```
$ npm install || yarn install
```
3. start the project
```
$ npm start
```

4. Open your localhost
It will automatically open a browser tab, but if for some reason it doesn't, you can go to
```
http://localhost:3000/
``` 

---

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+In this project we made a webapp where you can add a list of the books you've read
+
+---
+
+## Local installation
+
+1. Clone this repo
+```
+$ git clone git@github.com:YoseptF/library.git
+```
```
Comment (shloch) [inline README.md:16]: isn't it `git clone https://github.com/YoseptF/library.git` ??

File: index.html [hunk 2]
```diff
@@ -3,9 +3,33 @@
...
+        <form class="text-center" id="searchBook">
+          <label for="title">Title</label>
+          <input class="form-control mb-3" type="text" name="book[title]" id="title">
+          <label for="author">Author</label>
+          <input class="form-control mb-3" type="text" name="book[author]" id="author">
+          <label for="pages">Pages</label>
+          <input class="form-control mb-3" type="number" name="book[pages]" id="pages">
+          <label for="status">Aleady read</label>
+          <input class="form-control mb-3" type="checkbox" name="book[status]" id="status">
+          <input class="btn btn-primary btn-block btn-lg" type="submit" value="search">
```
Comment (shloch) [inline index.html:25]: should `value` not be instead `Add Book` ???

File: README.md (hunk 1 above)
Comment (YoseptF) [reply README.md:16]: They are both valid, the one we used in the README.md is this one
image
and you are using the https version
image

File: index.html (hunk 2 above)
Comment (YoseptF) [reply index.html:25]: hahaha, we noticed but forgot to changed before we sent it to review again, lemme change it, and then merge, thanks for the observation :D

**Pull request merged**